### DISCUSSION TITLE
Embedding Custom Modules

### DISCUSSION THREAD

**Discussion by tbugler70** [MasaCMS/MasaCMS#307] [General]
Core Version 7.4.6
App Server Lucee (5.4.1.8)
DB Server MySQL (8.1.0)

How does one go about embedding custom modules now? [m][/m] no longer works (as of 7.4 I beleive).

There nothing in the documentation and I wasn't able to find anything in the forums.

TIA

T.

**Comment by tbugler70** [comment]
Since the muratag removal was part of 7.4.6 (works on 7.4.5 and I don't seem to be able to enable it on 7.4.6) - I rolled back to 7.4.5. What is the plan moving forward? Are you getting rid of custom modules and everything should be built as a plugin? Just wondering what's the road map looking like.

**Reply by no3rdw** [reply]
You can re-enable the enableMuraTag setting and continue using mura tags, or use custom modules within any display region by making sure the module has a configurator.cfm file so it is able to be selected from the sidebar displayed while Inline Editing, and dragged to the page, or you can 'hardcode' your module into a site or component template by calling dspObject.

Some info on this from the Mura docs: https://docs.murasoftware.com/v7-1/mura-developers/mura-rendering/#section-mura-modules-display-objects

**Reply by tbugler70** [reply]
I did try re-enable mura tags in the settings config and did an app reload, but no dice. I'll give the configurator route a go.

Thanks
